pom.xml

```diff
@@ -452,10 +455,51 @@
                 <artifactId>commons-lang3</artifactId>
                 <version>3.12.0</version>
             </dependency>
+
+            <!-- Manage transitive dependency versions that have vulnerabilities reported -->
+            <dependency>
+                <groupId>org.xerial.snappy</groupId>
+                <artifactId>snappy-java</artifactId>
+                <version>1.1.10.5</version>
```

Comment:
Can we eventually pull all version numbers into properties. Not a blocker for this PR